Assets/Scripts/UI/SlipTimeUI.cs

```diff
@@ -1,33 +1,49 @@
-﻿using SlipTime;
+﻿using System;
+using SlipTime;
 using UnityEngine;
-using UnityEngine.Serialization;
 using UnityEngine.UI;
 
 namespace UI
 {
     public class SlipTimeUI : MonoBehaviour
     {
         public SlipTimeManager slipTimeManager;
-    
+
         public Text slipTimeUIText;
 
         private int slipTimeCharges;
 
         // Start is called before the first frame update
-        void Start()
+        private void Start()
         {
             slipTimeCharges = slipTimeManager.slipTimeCharges;
             slipTimeUIText.text = $"Slip Time Charges: {slipTimeCharges}";
         }
 
         // Update is called once per frame
-        void Update()
+        private void Update()
         {
-            if (slipTimeCharges != slipTimeManager.slipTimeCharges)
+            var slipTimeText = $"Slip Time Charges: {slipTimeManager.slipTimeCharges}";
+            
+            // Leave this code here for now. We may be able to re-use it later once we get a better UI for SlipTime
+            // if we don't have to constantly update the text box. If this comment is confusing, talk to NLow.
```

Comment:
Please do not leave usernames and user information in the comments, comment refactoring becomes difficult over time and we can always use Git Blame as a more efficient alternative. Leave a message in our messaging app if need be.

Reply:
Ok, removed.